AppAuth-JS-Ext-Logout/package-lock.json

```diff
@@ -1,70 +1,45 @@
 {
```

Comment:
This is generated source file. package-lock.json file is generated based on package.json to resolve dependencies. As standard, it is also committed